example_of_use_differential_evolution.ipynb

```diff
@@ -0,0 +1,148 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Example of use function - diferential evolution"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 45,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from scipy.optimize import differential_evolution\n",
+    "import numpy as np\n",
+    "from calculate_error import calculate_error "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### example parameters"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 48,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bounds = [[-4,4]] #example bounds\n",
```

Comment:
Szukamy tutaj ciągu o długości `len(true_val)`, więc `bounds` też musi być takiej długości. To działa rzecz jasna, ale dlatego, że `numpy` jest mądry i uznaje, że odejmujemy stałą od wszystkich wartości w tablicy. // Dla sklearn to by nie działało, ale znalazłabyś "błąd"(?) 

Jeżeli to działanie było zamierzone i celem było znalezienie liczby, która ma jak najmniejszą średnią odległość od danych punktów `true_val` to oczywiście zwracam honor. 🙏

Reply:
racja, poprawiłam bounds